clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/model/SubmitJobRequest.java

```diff
@@ -91,6 +100,9 @@ public String getConstraint() {
   private MigrationPolicy migrationPolicy = null;
   private DisruptionBudget disruptionBudget = null;
 
+  @JsonProperty("constraints")
```

Comment:
do you need this annotation given that you have one on line 428?

Reply:
yes that's not required , removed